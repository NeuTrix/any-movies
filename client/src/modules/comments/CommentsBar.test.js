import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
// import CommentsBar from './CommentsBar';

// Needs updating now that a container is wrapping
xdescribe('The CommentsBar component', () => {
	const props = {
		comments: [],
		count: 0,
		currMovie: {},
	};

	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<CommentsBar {...props} />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	describe('The CommentsBar snapshot', () => {
		const component = renderer.create(<CommentsBar {...props} />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
