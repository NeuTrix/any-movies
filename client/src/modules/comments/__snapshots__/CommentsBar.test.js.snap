// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="CommentsBar-root-5"
>
  <div
    className="MuiPaper-root-11 MuiPaper-elevation1-14 MuiPaper-rounded-12 MuiExpansionPanel-root-7 MuiExpansionPanel-rounded-8 CommentsBar-expansion-2"
  >
    <div
      aria-expanded={false}
      className="MuiButtonBase-root-44 MuiExpansionPanelSummary-root-38 CommentsBar-summary-6"
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <div
        className="MuiExpansionPanelSummary-content-42"
      >
        <p
          className="MuiTypography-root-47 MuiTypography-body2-55 CommentsBar-heading-3"
        >
          <span
            className="MuiBadge-root-83 CommentsBar-badge-1"
          >
            <span />
            <span
              className="MuiBadge-badge-84 MuiBadge-colorPrimary-85 MuiBadge-invisible-88"
            >
              0
            </span>
          </span>
          Be the first to review or reply!
        </p>
      </div>
      <div
        aria-hidden="true"
        className="MuiButtonBase-root-44 MuiIconButton-root-90 MuiExpansionPanelSummary-expandIcon-43"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={-1}
      >
        <span
          className="MuiIconButton-label-95"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-96"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-136"
        />
      </div>
    </div>
    <div
      aria-hidden="true"
      className="MuiCollapse-container-105"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper-107"
      >
        <div
          className="MuiCollapse-wrapperInner-108"
        >
          <button
            className="MuiButtonBase-root-44 MuiButton-root-109 MuiButton-outlined-117 MuiButton-outlinedPrimary-118"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="button"
          >
            <span
              className="MuiButton-label-110"
            >
              Add a Comment
            </span>
            <span
              className="MuiTouchRipple-root-136"
            />
          </button>
          <div
            className="MuiExpansionPanelDetails-root-135"
          />
          <div
            className="MuiExpansionPanelDetails-root-135 CommentsBar-expansion-2"
          >
            <div
              className="CommentsBar-list-4"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
